### ISSUE TITLE
Libreswan 4.1 fails to start in a Docker container

### ISSUE THREAD

**Issue by hwdsl2** [libreswan/libreswan#383]
Tested on Docker host: Ubuntu 20.04 and Debian 10
Docker image is [1] (based on Debian 10), with Libreswan updated from 3.32 to 4.1.

Error encountered:
```
Unknown init system sysvinit in use - aborted
```

I suspect that this issue may be triggered by commit 216e623 (or other recent commits in that file).

[1] https://github.com/hwdsl2/docker-ipsec-vpn-server

**Comment by letoams** [comment]
We use that for our testing.

Can you give me exact instructions on how you install/start or for me to reproduce ?

**Comment by hwdsl2** [comment]
@letoams Steps to reproduce the issue:
1. Install Docker on an Ubuntu 20.04 or Debian 10 system.
2. Download the project zip file from:
https://github.com/hwdsl2/docker-ipsec-vpn-server/archive/0026b9d722943a67a5b94e16704e8d7f2d917840.zip
and extract into a folder on your server.
3. Change into the new folder, build the Docker image:
```
docker build -t ipsec-vpn-server .
```
4. Run the Docker image:
```
docker run \
    --name ipsec-vpn-server-test \
    --restart=always \
    -p 500:500/udp \
    -p 4500:4500/udp \
    -d --privileged \
    ipsec-vpn-server
```
5. Check the container logs:
```
docker logs ipsec-vpn-server-test
```
You'll see the error:
```
Unknown init system sysvinit in use - aborted
```
If you want to check the Libreswan logs in this container, please see [1].

[1] https://github.com/hwdsl2/docker-ipsec-vpn-server#enable-libreswan-logs

**Comment by letoams** [comment]
The issue was "ipsec setup" was checking the used initsystem for "sysv" but the string it needed was "sysvinit"

Fix will be in libreswan 4.2. Thanks for the report and instructions on how to reproduce !
